pkg/cmd/server/start/start_master.go

```diff
@@ -451,7 +451,7 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 
 	var standaloneAssetConfig *origin.AssetConfig
 	if oc.WebConsoleEnabled() {
-		config, err := origin.BuildAssetConfig(*oc.Options.AssetConfig)
+		config, err := origin.NewAssetConfig(*oc.Options.AssetConfig, oc.Options.KubernetesMasterConfig.AdmissionConfig.PluginConfig)
```

Comment:
@deads2k @liggitt OK this is easy to back out if you don't like but I thought it would be good to pass in the config for all admission plugins rather than scope it to one. Briefly considered passing in the entire master config, decided on this scope for now.